scripts/setup-stripe-secrets.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+if [[ "$ENVIRONMENT" == "" ]]; then
+  echo "Must specify \$ENVIRONMENT to create stripe secret ">&2; exit 1;
+elif [[ "$ENVIRONMENT" == "stage" ]]; then
+PUBLISHABLE_API_KEY=$stagingStripePublicApiKey
+SECRET_API_KEY=$stagingStripeSecretApiKey
+elif [[ "$ENVIRONMENT" == "prod" ]]; then
+PUBLISHABLE_API_KEY=$productionStripePublicApiKey
+SECRET_API_KEY=$productionStripeSecretApiKey
+fi
+
+CLUSTER_NAME=${PROJECT_NAME}-${ENVIRONMENT}-${REGION}
+STRIPE_SECRET_NAME=${PROJECT_NAME}-${ENVIRONMENT}-stripe-${RANDOM_SEED}
+NAMESPACE=${PROJECT_NAME}
+
+API_KEY_JSON=$(printf '{"public_key":"%s","secret_key":"%s"}' "$PUBLISHABLE_API_KEY" "$SECRET_API_KEY");
+
+aws secretsmanager list-secrets --region $REGION --query "SecretList[?Tags[?Key=='stripe' && Value=='${PROJECT_NAME}-${ENVIRONMENT}']].[Name] | [0][0]"  \
+  || aws secretsmanager create-secret \
+  --region ${REGION} \
+  --name ${STRIPE_SECRET_NAME} \
+  --description "Stripe API Key" \
+  --tags "[{\"Key\":\"stripe\",\"Value\":\"${PROJECT_NAME}-${ENVIRONMENT}\"}]" \
+  --secret-string "${API_KEY_JSON}"
+
+
+BASE64_TOKEN=$(printf ${SECRET_API_KEY} | base64)
+## Modify existing application secret to have stripe api key
+kubectl --context $CLUSTER_NAME -n $NAMESPACE get secret ${PROJECT_NAME} -o json | \
```

Comment:
Is there even going to be an existing secret at this point? What is going to be creating it before this script runs?

Reply:
hmm youre right, i think we can remove the aws-secret entirely, originally i was thinking both frontend and backend can reference aws secret-manager, but it will still be annoying to update 
maybe just document the kubernetes secert / frontend's config file is the simplest solution

Reply:
Yeah let's do that for now. I think we're gonna need a new approach to some of the secret management stuff. I'll put some thought into it.